### DISCUSSION TITLE
Error: "Name ferr not defined."

### DISCUSSION THREAD

**Discussion by GoblinMode** [phoebe-project/phoebe2#573] [Questions]
I am trying to make a model of a detached binary star system by using the Phoebe web client. I put in as many parameters as possible into the client. However when I try to run the computation I get "Error name ferr not defined" any idea on how to fix this issue. I think it might be due to the fact that I have manually inputted my  known parameters (mass,distance,radius etc) but have not input a data set as I have my data in excel format which to my knowledge the client cannot read, so I have my light curve set to automatically generated.
I am very new to this software so any help is appreciated.

**Comment by kecnry** [comment]
Are you running a detached or remote job, by any chance?  I think I see where there might be a bug that is raising that error, but ultimately it's just preventing showing the real error.  If you run without detaching, you should be able to access the underlying error and I'll try to get a bugfix out for the `ferr` issue as soon as I get a chance.

Edit: I created #574 to track the `ferr` bug.

Thanks!
Kyle

**Comment by GoblinMode** [comment]
Hi Kyle
I am running it as a detached binary when I load up the web client if that
is what you mean by running detached/remote? So if I reload the
software and put all the parameters in as a contact system and run it again
i should get the real error?

**Reply by kecnry** [reply]
Ok, yes, the web client will always call the compute jobs as if they are detached, which is where that bug is being triggered and masking the true error statement from you.  Can you save the bundle and send it to me?  You can either attach it here or send it to me by email if you would prefer it not be public.

**Comment by GoblinMode** [comment]
Hi Kyle
Please find the attached bundle containing my input parameters below. I
tried attaching it in the github string but it wouldn't accept the data
type.

**Reply by kecnry** [reply]
It doesn't seem to be attached anywhere.  Can you please either send a link where its available to download or try emailing it to me (my email should be in my profile)?  Thanks!
